### PR TITLE
[NDS-703] new select with aria for v5

### PR DESCRIPTION
**Pull request by panvourtsis**
## Description

File changes
- most of the files are storyshots that changed due to the `role` attribute removal on the `ClickAwayListener` https://github.com/Orfium/orfium-ictinus/pull/647/files#diff-5447ae4490e60d047e53a9eee081d4a3ccc708bdf78fb514756f05acd2a33b90L31
- Also I changed the importation from files that were using `Select/Select` to import from the directory index `Select`, so you will see many

## Test Plan

- Keyboard navigation tested on select
- Storyshots per story

### CODE REVIEW

File: src/components/Select/Select.style.ts [hunk 1]
```diff
@@ -2,17 +2,11 @@ import { css, SerializedStyles } from '@emotion/react';
...
-      min-width: ${rem(150)};
-
-      * {
-        cursor: ${!isSearchable && 'pointer'};
-      }
-    `;
+export const selectWrapper = () => (): SerializedStyles =>
+  css`
+    position: relative;
+    min-width: ${rem(150)};
```
Comment (kostasdano) [inline src/components/Select/Select.style.ts:8]: make this a constant

File: src/components/List/NormalList/NormalList.tsx [hunk 2]
```diff
@@ -43,54 +42,74 @@ const NormalList = React.forwardRef<HTMLDivElement, NormalListProps>(
...
+          {...rest}
+          selectedKeys={
+            selectedOption !== undefined && selectedOption !== null
+              ? new Set([selectedOption.value])
+              : undefined
+          }
+          disabledKeys={
+            new Set(items.filter((option) => option.isDisabled).map((option) => option.value))
+          }
+          onSelectionChange={(keys) => {
```
Comment (kostasdano) [inline src/components/List/NormalList/NormalList.tsx:68]: maybe make this a named function (and all anonymous functions of this PR)

File: src/components/Select/Select.test.tsx [hunk 3]
```diff
@@ -61,6 +58,8 @@ describe('Generic Select', () => {
       selectInput = screen.getByPlaceholderText('Country') as HTMLInputElement;
       await selectDropdownOption(selectInput, dropdownList[1].label);
 
+      screen.debug();
+      debugger;
```
Comment (kostasdano) [inline src/components/Select/Select.test.tsx:62]: is this necessary?

File: src/components/Select/Select.tsx [hunk 4]
```diff
@@ -103,66 +37,103 @@ const Select = React.forwardRef<HTMLInputElement, SelectProps>((props, ref) => {
...
+            setTimeout(() => {
+              const firstChild = listRef.current?.firstChild;
+              if (firstChild instanceof HTMLElement && typeof firstChild.click === 'function') {
+                firstChild.click();
+              }
+            }, 0);
+          }
+        },
+        onBackspace: () => {
+          // isSearchable && onChange(isMulti ? [] : emptyValue);
```
Comment (kostasdano) [inline src/components/Select/Select.tsx:77]: comment

**Comment by kostasdano** [comment]
Screenshot_87

Loading dots are not rendering right

**Comment by kostasdano** [comment]
Screenshot_88

For some reason there are no options rendered when Select is clicked

**Comment by kostasdano** [comment]
select

when navigating through keyboard in the MultiSelect, if you click an option (with the help of ArrowDown and Enter) and then click BackSpace, the deleted option is highlighted when opening the SelectMenu. is this the expected behavior?

**Comment by kostasdano** [comment]
select2

When I click the X button on the Chip, I have to click twice in order for the deletion to be triggered. is this the expected behavior?

**Comment by panvourtsis** [comment]
> Screenshot_88
> 
> For some reason there are no options rendered when Select is clicked merging this 

this have been fixed by https://github.com/Orfium/orfium-ictinus/pull/655

**Comment by panvourtsis** [comment]
> select
> 
> when navigating through keyboard in the MultiSelect, if you click an option (with the help of ArrowDown and Enter) and then click BackSpace, the deleted option is highlighted when opening the SelectMenu. is this the expected behavior?

This has been resolved by #655

**Review by kostasdano — APPROVED** [review]
great work 🎉

**Pull request merged**